### ISSUE TITLE
Net Core samples?

### ISSUE THREAD

**Issue by albsilva** [Microsoft/vsts-dotnet-samples#93]
have any Net Core sample to show?

I try to run in Net Core 2.0 but is not working.

**Comment by MarkIannucci** [comment]
@albsilva ,  I've been doing some work with .Net Core 2.0.  I found that I needed to add the pre-release versions of the NuGet packages the documentation mentions.  Mine are:

- `Microsoft.TeamFoundationServer.Client (15.126.0-preview)`
- `Microsoft.VisualStudio.Services.Client (15.126.0-preview)`
- `Microsoft.VisualStudio.Services.InteractiveClient (15.126.0-preview)`

When I switched to those, a lot of the `using` statements referenced by their libraries stopped throwing exceptions.  Good luck.

**Comment by tiagosalgado** [comment]
@MarkIannucci do you have any sample of a csproj with those nuget packages? I've been trying to get those working with a .net core 2.0 project but no luck on getting those references working.

Thanks

**Comment by MarkIannucci** [comment]
@tiagosalgado see below:

```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
    <ApplicationInsightsResourceId>secret</ApplicationInsightsResourceId>
    <ApplicationInsightsAnnotationResourceId>another secret</ApplicationInsightsAnnotationResourceId>
  </PropertyGroup>

  <ItemGroup>
    <Folder Include="wwwroot\" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.2.1" />
    <PackageReference Include="Microsoft.ApplicationInsights.SnapshotCollector" Version="1.2.1-pre-20180313.2" />
    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.6" />
    <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="15.126.0-preview" />
    <PackageReference Include="Microsoft.VisualStudio.Services.Client" Version="15.126.0-preview" />
    <PackageReference Include="Microsoft.VisualStudio.Services.InteractiveClient" Version="15.126.0-preview" />
    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.0.3" />
    <PackageReference Include="Pdb2Pdb" Version="1.1.0-beta1-62316-01" />
  </ItemGroup>

  <ItemGroup>
    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.1" />
  </ItemGroup>

  <ItemGroup>
    <Content Update="appsettings.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
  </ItemGroup>

  <ItemGroup>
    <WCFMetadata Include="Connected Services" />
  </ItemGroup>

  <Import Project="shared project" Label="Shared" />

  <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
	<Exec Command="COPY /Y $(ProjectDir)..\appsettings.json $(ProjectDir) &amp; IF EXIST $(ProjectDir)appsettings.*.json (COPY /Y $(ProjectDir)appsettings.*.json $(TargetDir))" />
  </Target>

</Project>
```

**Comment by tiagosalgado** [comment]
Do you get all those nuget packages referenced properly? No matter what I do, I get 

![image](https://user-images.githubusercontent.com/1042389/37995746-5baa045e-320d-11e8-98db-663e10feb93b.png)

![image](https://user-images.githubusercontent.com/1042389/37995786-815f8d22-320d-11e8-86e2-7e41936f4424.png)

I get the same even using the last preview versions (15.131.0-preview).

**Comment by MarkIannucci** [comment]
Mine also have the yellow triangles and I get a few `NU1701` warnings saying that package may not be fully compatible with my project.

For example:
`Warning	NU1701	Package 'Microsoft.AspNet.WebApi.Client 5.2.2' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.0'. This package may not be fully compatible with your project.`

I think I've been able to handle the potential deployment issues because the VMs that are running this also have .Net 4.6.1 installed...

**Comment by tiagosalgado** [comment]
My problem is before even deploying anywhere.

![image](https://user-images.githubusercontent.com/1042389/37997114-ee1f911a-3211-11e8-865c-f71a8a4b7f5d.png)

Guess until the client libraries are fully supported by .net core, will face this issue.

**Comment by Grandizer** [comment]
I would agree it was a duplicate.  I would just love a status/state of where things are for a full .Net Core version?  Or at least a guestimate of when it will be released.  Thank you.

**Comment by david-beckman** [comment]
It looks like v16 supports .NET Standard 2.0:
[16.143.1](https://www.nuget.org/packages/Microsoft.TeamFoundationServer.Client/16.143.1)
Might be possible now!
